tests/cpp/test_host_irs.cpp

```diff
@@ -487,6 +487,33 @@ TEST_F(NVFuserTest, HostIrSetStream) {
       c10::cuda::getDefaultCUDAStream(0), c10::cuda::getCurrentCUDAStream(0));
 }
 
+// The following test simply demonstrate how to change current CUDA stream in
+// the host program
+TEST_F(NVFuserTest, HostIrDefaultStream) {
```

Comment:
This and the previous test should probably avoid using NVFuserTest as the fixture **name** -- NVFuserTest has been too overloaded despite that many legacy tests in test_gpu*.cpp still use it. 

One possibility is
```
TEST_F(HostIrTest, DefaultStream) {
```
Even though `HostIrTest` has parameters, it's OK to use it in TEST_F as long as GetParam isn't called. 

(You can use a different fixture name that you like. It's really just NVFuserTest I wanted to avoid.)

Reply:
> Even though HostIrTest has parameters, it's OK to use it in TEST_F as long as GetParam isn't called.

Doing this, I'm getting an error:

```
In file included from /opt/pytorch/Fuser/third_party/googletest/googletest/include/gtest/gtest-printers.h:122,
                 from /opt/pytorch/Fuser/third_party/googletest/googletest/include/gtest/gtest-matchers.h:49,
                 from /opt/pytorch/Fuser/third_party/googletest/googletest/include/gtest/internal/gtest-death-test-internal.h:47,                                                             from /opt/pytorch/Fuser/third_party/googletest/googletest/include/gtest/gtest-death-test.h:43,
                 from /opt/pytorch/Fuser/third_party/googletest/googletest/include/gtest/gtest.h:65,
                 from /opt/pytorch/Fuser/tests/cpp/test_host_irs.cpp:8:                                                                                                      /opt/pytorch/Fuser/tests/cpp/test_host_irs.cpp:480:1: error: ‘nvfuser::hir::StreamHostIrTestParams_HostIrSetStream_Test::~StreamHostIrTestParams_HostIrSetStream_Test()’ mark
ed ‘override’, but does not override
  480 | TEST_F(StreamHostIrTestParams, HostIrSetStream) {
      | ^~~~~~
```

Let me use a different class.